mmcv/onnx/simplify/common.py

```diff
@@ -0,0 +1,43 @@
+import copy
+import warnings
+
+import onnx
+
+
+def add_suffix2name(ori_model, suffix='__', verify=True):
```

Comment:
It seems that this method is only used once. Can it be merged into `core.py`?

Reply:
The intention is that code by mmcv should be added into `common.py`.
While `core.py` would be kept updated with originally [onnx-simplifier](https://github.com/daquexian/onnx-simplifier).